### ISSUE TITLE
Can we make strict window title matching a per-overlay setting?

### ISSUE THREAD

**Issue by Identifier** [elvissteinjr/DesktopPlus#58]
I'm using the NewUI branch, and I'm able to turn off strict title matching in order to get Discord to work in an overlay.  (Discord's title changes based on the channel you're in).  But unfortunately having strict title matching off means other apps with multiple windows doesn't work anymore.  (For example, the Elite Dangerous Discovery app can have multiple different windows open at the same time, with different titles, but all from EDDiscovery.exe and with the same window class name.).  

Anyway, long story short, can we move the option for strict title matching to a per-overlay setting instead of all overlays?  Then when doing window-to-overlay-matching, match all the strict ones first before matching the non-strict ones to the remaining windows.

**Comment by elvissteinjr** [comment]
As with the other issue, uh, sorry for the late response.
Just to not leave this hanging until I've handled it, I do agree it would make a lot of sense. 
I'll change the setting to be an overlay property instead.

**Comment by elvissteinjr** [comment]
Strict window matching is now a per-overlay setting. In the code at least (not rushing to do another preview build right now).

I'll close this for now, but let me know if there's further issues with this. Thanks.
